app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def login user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    current_user || User.find_by(id: session[:user_id])
```

Comment:
chỗ này code chưa chuẩn

Reply:
em có xem code của các b phần này: 
> current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
nhưng kh pass rubocop, vẫn lỗi Useless assignment to local variable.
> current_user || User.find_by(id: session[:user_id]) if session[:user_id]
em chỉ bố sung điều kiện if thì pass rubocop ạ